pkgs/servers/monitoring/grafana/default.nix

```diff
@@ -99,9 +104,16 @@ buildGoModule (finalAttrs: {
     wire gen -tags oss ./pkg/server
     wire gen -tags oss ./pkg/cmd/grafana-cli/runner
 
-    GOARCH= CGO_ENABLED=0 go generate ./kinds/gen.go
-    GOARCH= CGO_ENABLED=0 go generate ./public/app/plugins/gen.go
-
+    # Does not check CC_FOR_BUILD or LD_FOR_BUILD:
+    # https://github.com/golang/go/blob/b194f5d24a71e34f147c90e4351d80ac75be55de/src/cmd/cgo/gcc.go#L1724
+    # ```
+    # go-1.25.4/share/go/pkg/tool/linux_amd64/link: running aarch64-unknown-linux-gnu-gcc failed: exit status 1
+    # aarch64-unknown-linux-gnu-gcc -m64 -s -o $WORK/b001/exe/gen -rdynamic /build/go-link-507658645/go.o
+    # aarch64-unknown-linux-gnu-gcc: error: unrecognized command-line option '-m64'
+    # ```
+    # Above log is due to https://github.com/golang/go/blob/b194f5d24a71e34f147c90e4351d80ac75be55de/src/cmd/cgo/gcc.go#L1763
+    CC=$CC_FOR_BUILD LD=$CC_FOR_BUILD GOARCH= CGO_ENABLED=0 go generate ./kinds/gen.go
+    CC=$CC_FOR_BUILD LD=$LD_FOR_BUILD GOARCH= CGO_ENABLED=0 go generate ./public/app/plugins/gen.go
```

Comment:
I tested and the CGO_ENABLED=0 is not required and gets rid of a warning
It was added in https://github.com/nixos/nixpkgs/commit/78b312a895f8368746318bacb2634a151e8ea6ae

This whole issue may have happened because go isn't respecting `CGO_ENABLED=0` fully anymore